build.gradle

```diff
@@ -1,34 +1,45 @@
 plugins {
-    id "com.jfrog.bintray" version "1.1"
-    id "com.github.jruby-gradle.base" version "0.1.5"
     id "java"
     id "checkstyle"
     id "jacoco"
-    id "findbugs"
+    id "maven-publish"
+    id "org.embulk.embulk-plugins" version "0.4.2"
 }
-import com.github.jrubygradle.JRubyExec
 repositories {
     mavenCentral()
     jcenter()
 }
-configurations {
-    provided
-}
 
+group = "io.github.sonots"
 version = "0.5.1"
-sourceCompatibility = 1.7
-targetCompatibility = 1.7
+description = "A filter plugin for Embulk to filter out rows with conditions."
+
+sourceCompatibility = 1.8
+targetCompatibility = 1.8
 
 dependencies {
-    compile  "org.embulk:embulk-core:0.8.+"
-    provided "org.embulk:embulk-core:0.8.+"
+    compileOnly "org.embulk:embulk-api:0.10.30"
+    compileOnly "org.embulk:embulk-spi:0.10.30"
+    compile("org.embulk:embulk-util-config:0.3.0") 
+    compile "org.embulk:embulk-util-timestamp:0.2.1"
+    compile "org.embulk:embulk-util-text:0.1.1"
+    compile "org.embulk:embulk-util-timestamp:0.2.1"
+    compile('org.embulk:embulk-util-json:0.1.0') {
```

Comment:
`embulk-util-json:0.1.1` won't need excluding `msgpack-core`.

Reply:
Thanks. Update to 0.1.0 -> 0.1.1